packages/ethers-5/src/client.ts

```diff
@@ -0,0 +1,788 @@
+import type {
+  CaipAddress,
+  CaipNetwork,
+  CaipNetworkId,
+  ConnectionControllerClient,
+  Connector,
+  LibraryOptions,
+  NetworkControllerClient,
+  PublicStateControllerState,
+  Token
+} from '@web3modal/scaffold'
+import { Web3ModalScaffold } from '@web3modal/scaffold'
+import {
+  ADD_CHAIN_METHOD,
+  COINBASE_CONNECTOR_ID,
+  EIP6963_ANNOUNCE_EVENT,
+  EIP6963_CONNECTOR_ID,
+  EIP6963_REQUEST_EVENT,
+  INJECTED_CONNECTOR_ID,
+  NAMESPACE,
+  VERSION,
+  WALLET_CONNECT_CONNECTOR_ID,
+  WALLET_ID,
+  ConnectorExplorerIds,
+  ConnectorImageIds,
+  ConnectorNamesMap,
+  ConnectorTypesMap,
+  NetworkBlockExplorerUrls,
+  NetworkImageIds,
+  NetworkNames,
+  NetworkRPCUrls,
+  networkCurrenySymbols
+} from '@web3modal/utils'
+import type EthereumProvider from '@walletconnect/ethereum-provider'
+import {
+  caipNetworkIdToNumber,
+  getEthersCaipDefaultChain,
+  getCaipTokens,
+  hexStringToNumber
+} from '@web3modal/utils'
+import type { Address, ProviderType } from './utils/types.js'
+import { ethers, utils } from 'ethers'
+import { ProviderController } from './store/index.js'
```

Comment:
Let's keep same patterns going and swap `store/index.js` to `controllers/ProviderController.js`